apps/src/feedback.js

```diff
@@ -529,20 +515,10 @@ FeedbackUtils.prototype.displayFeedback = function (
   }
 };
 
-function logDialogActions(event, options, isPerfect) {
-  if (experiments.isEnabled(experiments.FINISH_DIALOG_METRICS)) {
-    firehoseClient.putRecord({
-      study: FIREHOSE_STUDY,
-      study_group: dialog_type,
-      event: event,
-      data_json: JSON.stringify({
-        level_type: options.level ? options.level.skin : null,
-        level_id: options.response ? options.response.level_id : null,
-        level_path: options.response ? options.response.level_path : null,
-        isPerfectBlockCount: isPerfect,
-      }),
-    });
-  }
+function recordFinishShare(type, appType) {
+  analyticsReporter.sendEvent(EVENTS.type, {
```

Comment:
Does this work? I thought you had to use bracket syntax like EVENTS[type] when using a variable as a key.

Reply:
Thanks Ben! 🙏  Just updated similar to how I updated the events for the 'share' button. 
I think I'm probably working on too many branches simultaneously rn!